src/dask_awkward/lib/structure.py

```diff
@@ -319,10 +314,8 @@ def combinations(
 
 
 @borrow_docstring(ak.copy)
-def copy(array):
-    raise DaskAwkwardNotImplemented(
-        "This function is not necessary in the context of dask-awkward."
-    )
+def copy(array: Array) -> Array:
+    return array
 
```

Comment:
This could be a little confusing, no? Folks might expect to get a new object

Reply:
This change should have been louder and/or had a review from you, sorry.

My rationale is that we want `ak.XXX` to work on both `ak` and `dak` `Array` objects. Thus, we should still support the function. However, I now realise that with the way that coffea (ab)uses `behavior`, and now with `attrs`, we'll need to perform a shall copy. Let me make a new PR.

Otherwise, are you happy with this rationale?